docs/networking.adoc

```diff
@@ -191,8 +191,8 @@ underlay0   vnic      9000   up       --         underlay_stub0
 We see now that we have:
 
 * the three physical devices mentioned earlier
-* an etherstub `underlay_stub0` with a VNIC `underlay0` over it
-* an etherstub `bootstrap_stub0` with a VNIC `boostrap0` over it
+* an etherstub `underlay_stub0` with a VNIC `underlay0` over it. Every underlay# vnic can only connect to other underlay# vnics on the same sled.
+* an etherstub `bootstrap_stub0` with a VNIC `boostrap0` over it. Every bootstrap# vnic can only connect to other bootstrap# vnics on the same sled.
```

Comment:
```suggestion
* an etherstub `underlay_stub0` with a VNIC `underlay0` over it. Every underlayN VNIC can only connect to other underlayN VNICs on the same sled.
* an etherstub `bootstrap_stub0` with a VNIC `boostrap0` over it. Every bootstrapN VNIC can only connect to other bootstrapN VNICs on the same sled.
```

But I think this is more confusing than helpful.  In a sense, the underlayN VNICs are only virtually "physically" connected to the other underlayN VNICs.  But at the IP level, they have connectivity to the underlay interfaces on other sleds.  So I think someone might read that sentence and think "the underlay interfaces aren't connected to other sleds", which is wrong.

Reply:
Even though the **Data path between components on different sleds** section lower in the document makes it clear that the gz's underlay0 will forward the packet, it seems like it might help to state it here as well.

Perhaps something like a new paragraph: just below:

_Each stub network by itself is only able to communication between vnic's in the respective stub network unless port forwarding is enabled in the global zone, which it is._

or 

an etherstub `underlay_stub0` with a VNIC `underlay0` over it. Every bootstrapN VNIC can only connect to other bootstrapN VNICs on the same sled unless the global zone is setup to forward packets, which it is.

Reply:
That still reads pretty misleading to me.  It's not until you get to end of the sentence that you realize the whole sentence is actually irrelevant for the use case we're writing for.

We could add: "The underlayN VNICs are only directly connected to each other.  They rely on IP-level forwarding (explained shortly) to provide connectivity beyond the etherstub network." and the same thing for the bootstrapN VNICs.

I'd prefer not to unless you or somebody else found this point confusing though.  The text already said this above, and there's nothing in the output here that reinforces that point.  I think it's potentially more confusing to sort of pre-introduce the IP-level concerns before we really get to them.  Not a big deal either way though.